Tests/BacktraceMetricsTest.swift

```diff
@@ -16,35 +16,31 @@ final class BacktraceMetricsTests: QuickSpec {
 
             let summedEventName = "view-changed"
             let uniqueEventName = "guid"
-
-            it("clears the summed event after enabling and sending") {
-                let metrics = BacktraceMetrics(api: backtraceApi)
-                metrics.enable(settings: BacktraceMetricsSettings())
-
-                // Allow default events to be "sent" out
-                expect { metrics.count }.toEventually(equal(1), timeout: .seconds(5), pollInterval: .milliseconds(100))
-            }
-
-            it("can add and store summed event") {
-                let metrics = BacktraceMetrics(api: backtraceApi)
-                metrics.enable(settings: BacktraceMetricsSettings())
-
-                // Allow default events to be "sent" out
-                expect { metrics.count }.toEventually(equal(1), timeout: .seconds(5), pollInterval: .milliseconds(100))
-
-                metrics.addSummedEvent(name: summedEventName)
-                expect { metrics.count }.to(equal(2))
-            }
-
-            it("can add and store unique event") {
-                let metrics = BacktraceMetrics(api: backtraceApi)
-                metrics.enable(settings: BacktraceMetricsSettings())
-
-                // Allow default events to be "sent" out
-                expect { metrics.count }.toEventually(equal(1), timeout: .seconds(5), pollInterval: .milliseconds(100))
-
-                metrics.addUniqueEvent(name: uniqueEventName)
-                expect { metrics.count }.to(equal(2))
+            
+            var metrics: BacktraceMetrics?
+            
+            context("Events operation") {
+                
+                beforeEach {
+                    metrics = BacktraceMetrics(api: backtraceApi)
+                    metrics?.enable(settings: BacktraceMetricsSettings())
+                    metrics?.clearSummedEvents()
+                }
+                
+                it("clears the summed event after enabling and sending") {
+                    // Allow default events to be "sent" out
+                    expect { metrics?.count }.toEventually(equal(1))
+                }
+                
+                it("can add and store summed event") {
+                    metrics?.addSummedEvent(name: summedEventName)
+                    expect { metrics?.count }.to(equal(2))
```

Comment:
I think the test is about to verify if the number of tests added to the queue will increase. I think that's the better way to verify that. We shouldn't think about hard coded numbers but rather about the behavior
```suggestion
                    let count =  metrics?.count
                    metrics?.addSummedEvent(name: summedEventName)
                    expect { metrics?.count }.to(equal(count + 1))
```

Also maybe it would be better to verify if the summedEventName is in the metrics list?